data/assets/scene/solarsystem/planets/jupiter/minor/ananke_group.asset

```diff
@@ -207,22 +207,61 @@ local anankeGroup = {
   }
 }
 
+--Generate all moon labels
+local moon_labels = {}
+
+for i, moon in ipairs(anankeGroup) do
+  moon_labels[i] = {
+    Identifier = moon.Identifier .. "Label",
+    Parent = moon.Identifier,
+    Renderable = {
+      Enabled = false,
+      Type = "RenderableLabel",
+      Text = moon.Identifier,
+      FontSize = 70.0,
+      Size = 7.1,
+      MinMaxSize = { 1, 25 },
+      OrientationOption = "Camera View Direction",
+      BlendMode = "Additive",
+      EnableFading = true,
+      FadeUnit = "Gm",
+      FadeDistances = { 1.5, 90.0 },
+      FadeWidths = { 0.5, 90.0 }
+    },
+    Tag = { "solarsystem_labels", "moon_labels", "minor_moon_labels" },
+    GUI = {
+      Name = moon.Identifier .. " Label",
+      Path = "/Solar System/Planets/Jupiter/Moons",
+      Description = "Label for Jupiter's moon " .. moon.Identifier .. " (Ananke group)"
```

Comment:
Should this be `moon.GUI.Name` instead?  In most cases those would be the same, but they don't have to be

Reply:
Same for a bunch of other places

Reply:
not all moons have GUI name set it seems (see image below).
Should I update all assets that are missing this property?
Or should I do a conditional that tries to get moon.GUI.name but have fallback for Identifier?

![moon_gui](https://user-images.githubusercontent.com/9076357/215431721-69086449-9f3e-49ce-abfc-16afbd97e8ab.png)

Reply:
Note:
It was decided that conditional will be implemented.